instrumentation/mongo/lib/opentelemetry/instrumentation/mongo/middlewares/subscriber.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require_relative 'command_serializer'
+
+module OpenTelemetry
+  module Instrumentation
+    module Mongo
+      module Middlewares
+        # Event handler class for Mongo Ruby driver
+        class Subscriber
+          THREAD_KEY = :__opentelemetry_mongo_spans__
+
+          def started(event) # rubocop:disable Metrics/AbcSize
+            # start a trace and store it in the current thread; using the `operation_id`
+            # is safe since it's a unique id used to link events together. Also only one
+            # thread is involved in this execution so thread-local storage should be safe. Reference:
+            # https://github.com/mongodb/mongo-ruby-driver/blob/master/lib/mongo/monitoring.rb#L70
+            # https://github.com/mongodb/mongo-ruby-driver/blob/master/lib/mongo/monitoring/publishable.rb#L38-L56
+            # TODO: decide naming convention
+            collection = get_collection(event.command)
+            span = tracer.start_span("#{collection || ''}.#{event.command_name}", kind: :client)
```

Comment:
Generally I think this fits the specification per https://github.com/open-telemetry/opentelemetry-specification/blob/acd6f771d6064b92738a96afd729c59ba8ab9c4d/specification/trace/semantic_conventions/database.md#mongodb

That being said it looks like: 

- python has a slightly different approach here, with `mongodb.<collection>.<operation>` https://github.com/open-telemetry/opentelemetry-python/blob/f3cdfa2cdb9cb1c442189e2ead3788f45d92352d/instrumentation/opentelemetry-instrumentation-pymongo/src/opentelemetry/instrumentation/pymongo/__init__.py#L66

- and js is even less descriptive but also prepends the name with `mongodb` https://github.com/open-telemetry/opentelemetry-js-contrib/blob/088a4d57afa7a36c8f86c9afaa82d18477c82e5c/plugins/node/opentelemetry-plugin-mongodb/src/mongodb.ts#L232 

whereas ruby is just `<collection>.<operation>`. It seems like it's not part of the spec but it couldn't hurt  to prepend with `mongodb`

Reply:
I agree to prepend with mongodb to be consistent with python.